python/paddle/fluid/layers/learning_rate_scheduler.py

```diff
@@ -32,14 +32,40 @@
 """
 
 
-def _decay_step_counter():
+def _decay_step_counter(begin=0):
     # the first global step is zero in learning rate decay
     global_step = nn.autoincreased_step_counter(
-        counter_name='@LR_DECAY_COUNTER@', begin=0, step=1)
+        counter_name='@LR_DECAY_COUNTER@', begin=begin, step=1)
     global_step = tensor.cast(global_step, 'float32')
     return global_step
 
 
+def nmt_nist_decay(d_model, warmup_steps):
```

Comment:
if this is not a general decay method, maybe we should put it in transformer model?

Reply:
Done.Thanks!